provider/osio/osio.go

```diff
@@ -166,13 +175,23 @@ func (p *Provider) loadRules(clusterResp *clusterResponse) *types.Configuration
 }
 
 func createFrontend(clusterURL string, backend string) *types.Frontend {
+	clusterURL = normalizeURL(clusterURL)
 	routes := make(map[string]types.Route)
-	routes["test_1"] = types.Route{Rule: "HeadersRegexp:Target," + clusterURL}
+	routes["test_1"] = types.Route{Rule: "Headers:Target," + clusterURL}
```

Comment:
This will match absolute target_url rather regex. This change is made for better performance.